listener/types.go

```diff
@@ -19,23 +19,13 @@ import (
 
 type EthBlock struct {
 	block *ethtypes.Block
-	txs   []bridgeCore.Transaction
 	logs  []bridgeCore.Log
 }
 
-func NewEthBlock(client utils.EthClient, chainId *big.Int, block *ethtypes.Block, getLogs bool) (*EthBlock, error) {
+func NewEthBlock(client utils.EthClient, block *ethtypes.Block, getLogs bool) (*EthBlock, error) {
 	ethBlock := &EthBlock{
 		block: block,
 	}
-	// convert txs into ILog
-	for _, tx := range block.Transactions() {
-		transaction, err := NewEthTransaction(chainId, tx)
-		if err != nil {
-			log.Error("[NewEthBlock] error while init new Eth Transaction", "err", err, "tx", tx.Hash().Hex())
-			return nil, err
-		}
-		ethBlock.txs = append(ethBlock.txs, transaction)
-	}
 	if getLogs {
```

Comment:
I think we should remove this getLogs logic too, it is unused and costs 1 round of RPC request